docs/development/design/apiv3.rst

```diff
@@ -1,15 +1,34 @@
-Design of APIv3
-===============
+APIv3
+=====
+
+This document describes the design,
+some decisions already made and an implementation plan for APIv3 in stages.
 
 APIv3 will be designed to be easy to use and useful to perform read and write operations as the main two goals.
 
 It will be based on Resources as APIv2 but considering the ``Project`` resource as the main one,
 from where most of the endpoint will be based on it.
 
 
+Goals
+-----
+
+* Easy to use for our users
+* Useful to perform read and write operations
+* Cover most useful cases
```

Comment:
This needs to be clearer. We should define these.

Reply:
I expand the list. Looks better now and more concise.